client/post-editor/edit-post-status/index.jsx

```diff
@@ -53,7 +60,9 @@ var EditPostStatus = React.createClass( {
 		stats.recordStat( stickyStat );
 		stats.recordEvent( 'Changed Sticky Setting', stickyEventLabel );
 
+		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
```

Comment:
I totally agree.
But in an effort to keep this PR mergeable, I'm not touching previous behaviour right now.
In my opinion this whole code should be moved to redux action and analytics handled by middleware